app/vmui/packages/vmui/src/components/Configurators/QueryEditor/QueryEditorAutocomplete.tsx

```diff
@@ -176,6 +192,7 @@ const QueryEditorAutocomplete: FC<QueryEditorAutocompleteProps> = ({
 
   return (
     <>
+      {metric}
```

Comment:
Is this needed here? It looks weird in UI
<img width="306" alt="image" src="https://github.com/user-attachments/assets/a86fa276-c855-4165-9f09-4f7ac725d94d">